FEATURES.md

```diff
@@ -88,7 +88,7 @@ If your PR makes notable changes to Winit's features, please update this section
 - **Popup / modal windows**: Windows can be created relative to the client area of other windows, and parent
   windows can be disabled in favor of popup windows. This feature also guarantees that popup windows
   get drawn above their owner.
-
+- **Window centering**: Sets the window position to be in the center of the given monitor.
```

Comment:
If you add an entry here, you need to also add it to the compatibility matrix below.